patched_bdist_wheel.py

```diff
@@ -0,0 +1,103 @@
+"""
```

Comment:
this would actually be used by all people on windows, not only CI: If I understand that bugreport right, then this patch will be needed if you want to build a wheel on windows... Not sure where this should go, tools? If it's in a subdir, it also needs a `__init__.py`?